KarhooUISDK/CommonUI/Views/SeparatorWithLabelView/SeparatorWithLabelView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  SeparatorWithLabelView.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 07.02.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import UIKit
+
+struct KHSeparatorWithLabelViewID {
+    public static let containerStackView = "container_stack_view"
+    public static let leftSeparator = "left_separator_view"
+    public static let rightSeparator = "right_separator_view"
+    public static let label = "label"
+}
+
+final class SeparatorWithLabelView: UIView {
+    private lazy var leftSeparator = UIView().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.backgroundColor = KarhooUI.colors.border
+        $0.accessibilityIdentifier = KHSeparatorWithLabelViewID.leftSeparator
+    }
+    
+    private lazy var rightSeparator = UIView().then {
+        $0.translatesAutoresizingMaskIntoConstraints = false
+        $0.backgroundColor = KarhooUI.colors.border
+        $0.accessibilityIdentifier = KHSeparatorWithLabelViewID.rightSeparator
+    }
```

Comment:
In CommonUI is LineView class and in rest of code separators have that type.

Reply:
Done